app/controllers/training_modules_controller.rb

```diff
@@ -26,6 +26,7 @@ def certificate
       track('module_complete')
       mod_time = ModuleTimeToComplete.new(user: current_user, training_module_id: params[:training_module_id])
       mod_time.update_time
+      track('time_taken_to_complete_module', time_in_seconds: mod_time.result)
```

Comment:
I feel this is more an additional attribute on the `module_complete` event, rather than a wholly separate key.